micrometer-shim/src/main/java/io/opentelemetry/micrometershim/Bridging.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.micrometershim;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.Statistic;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+
+final class Bridging {
+
+  static Attributes tagsAsAttributes(Meter.Id id, NamingConvention namingConvention) {
```

Comment:
Removed caches from this class since we don't (yet) have a concurrent cache in this repo

- I've always wondered whether we should cache `AttributesBuilder.put(String key` invocations in the API itself. I don't think there's anything unique to micrometer to do it specifically here
- #4222 will land soon to remove need for description cache

Reply:
> * I've always wondered whether we should cache `AttributesBuilder.put(String key` invocations in the API itself. I don't think there's anything unique to micrometer to do it specifically here

That would be awesome, I think - we've been recommending using `AttributeKey` everywhere to avoid additional allocations, but if we could fix that at the source that would be great.

Reply:
I'm a bit hesitant, only because we'd need to be careful about how much we cache... we wouldn't want a pathological API usage to OOM us.